index.js

```diff
@@ -23,6 +23,11 @@ exports.HOSTS = WINDOWS
  * @param  {function(err, lines)=} cb
  */
 
+exports.setFile = function(filePath)
+{
+  exports.HOSTS = filePath
+}
+
```

Comment:
As the app runs per instance this should be reset every time it's run unless I'm missing something.